cds_ils/migrator/eitems/api.py

```diff
@@ -233,33 +236,44 @@ def migrate_ebl_links():
     search = get_documents_with_ebl_eitems()
     click.echo("Found {} documents with ebl links.".format(search.count()))
 
+    url_template = \
+        "http://ebookcentral.proquest.com/lib/cern/detail.action?docID={}"
+
     for hit in search.scan():
         # make sure the document is in DB not only ES
         document = Document.get_record_by_pid(hit.pid)
```

Comment:
Please do not hardcode class names :) Can you double check everywhere in the code?